bot/exts/holidays/valentines/be_my_valentine.py

```diff
@@ -30,7 +32,7 @@ def load_json() -> dict:
         return loads(p.read_text("utf8"))
 
     @in_month(Month.FEBRUARY)
-    @commands.group(name="lovefest")
+    @commands.command(name="lovefest", help=f"NOTE: This command has been moved to {MOVED_COMMAND}")
```

Comment:
Could you please move this help string to the command's docstring. Our help command uses those to populate the help text.

Reply:
The reason its here is because its dynamically using the configured prefix of the other bot. Additionally, this is used for the help command even in this location.

Reply:
Setting help in the command decorator overrides the callback docstring, as shown below

![image](https://user-images.githubusercontent.com/71233171/144685266-10aa6528-ad4f-4957-b9f7-dab6b684ed0e.png)
![image](https://user-images.githubusercontent.com/71233171/144685288-2712c695-5e9a-4b26-a0d7-08760afeb1c7.png)

Reply:
Ahh alright cool, needing dynamic help here is a good reason. Could you hosted the docstring to reflect the new behaviour? It still references what it used to do.

Reply:
cfb2cc69